examples/Introducing_CivisML_v2.ipynb

```diff
@@ -956,13 +965,15 @@
    "metadata": {},
    "source": [
     "## Model Portability\n",
-    "What if you want to score a model outside of Civis Platform? Maybe you want to deploy this model in an app for education policy makers. In CivisML v2, you can easily get the trained model out of the `ModelFuture` object."
+    "What if you want to score a model outside of Civis Platform? Maybe you want to deploy this model in an app for education policy makers. In CivisML 2.0, you can easily get the trained model pipeline out of the `ModelFuture` object."
```

Comment:
I'm not sure the addition of the word "pipeline" adds anything here. "model" should be able to stand by itself.

Reply:
Marshall suggested this change to clarify that it includes pre-processing, which the user might not automatically think of as part of the model. I don't have strong feelings about it either way.

Reply:
If I didn't know already what was going on, I'm not sure this would help, but I also don't have strong feelings.